README.md

```diff
@@ -207,6 +224,31 @@ Examples:
 </pre>
 ```
 
+
+##### separator
```

Comment:
I think this section is a bit redundant and can be removed since the section above already captures usage of the `separator` prop.